rebar.config

```diff
@@ -4,10 +4,8 @@
 
 {profiles, [
     {test, [{deps, [
-        {proper, ".*",
-            {git, "git://github.com/manopapad/proper.git", {tag, "v1.2"}}},
-        {cth_readable, ".*",
-            {git, "https://github.com/ferd/cth_readable.git", {ref, "0fc53d1"}}}
+        {proper, "1.3.0"},
+        {cth_readable, "1.4.2"}
```

Comment:
rebar3 now ships with cth_readable so you likely no longer need to include the lib and add the hook by hand